CRITICAL - 2021-07-20 15:37:58 --> Invalid argument supplied for foreach()
#0 D:\xampp\htdocs\ushno-app\app\Views\admin\pages\horario.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'D:\\xampp\\htdocs...', 23, Array)
#1 D:\xampp\htdocs\ushno-app\system\View\View.php(220): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\ushno-app\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\ushno-app\system\Common.php(1217): CodeIgniter\View\View->render('admin/pages/hor...', Array, true)
#4 D:\xampp\htdocs\ushno-app\app\Controllers\HorarioController.php(50): view('admin/pages/hor...', Array)
#5 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(936): App\Controllers\HorarioController->index()
#6 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorarioController))
#7 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\ushno-app\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-07-20 15:38:02 --> Invalid argument supplied for foreach()
#0 D:\xampp\htdocs\ushno-app\app\Views\admin\pages\horario.php(23): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Invalid argumen...', 'D:\\xampp\\htdocs...', 23, Array)
#1 D:\xampp\htdocs\ushno-app\system\View\View.php(220): include('D:\\xampp\\htdocs...')
#2 D:\xampp\htdocs\ushno-app\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#3 D:\xampp\htdocs\ushno-app\system\Common.php(1217): CodeIgniter\View\View->render('admin/pages/hor...', Array, true)
#4 D:\xampp\htdocs\ushno-app\app\Controllers\HorarioController.php(50): view('admin/pages/hor...', Array)
#5 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(936): App\Controllers\HorarioController->index()
#6 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorarioController))
#7 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\ushno-app\public\index.php(44): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-07-20 15:40:48 --> You must set the database table to be used with your query.
#0 D:\xampp\htdocs\ushno-app\system\Model.php(615): CodeIgniter\Database\BaseConnection->table('')
#1 D:\xampp\htdocs\ushno-app\system\Model.php(834): CodeIgniter\Model->builder()
#2 D:\xampp\htdocs\ushno-app\app\Controllers\HorarioController.php(151): CodeIgniter\Model->__call('addHorario', Array)
#3 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(936): App\Controllers\HorarioController->save()
#4 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HorarioController))
#5 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xampp\htdocs\ushno-app\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-07-20 19:40:42 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 19:53:34 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 19:54:19 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 22:11:36 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 22:32:42 --> syntax error, unexpected 'resources' (T_STRING), expecting ')'
#0 D:\xampp\htdocs\ushno-app\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 D:\xampp\htdocs\ushno-app\system\Common.php(1217): CodeIgniter\View\View->render('emisora/pages/i...', Array, true)
#2 D:\xampp\htdocs\ushno-app\app\Controllers\HomeController.php(13): view('emisora/pages/i...')
#3 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(936): App\Controllers\HomeController->index()
#4 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#5 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xampp\htdocs\ushno-app\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-07-20 22:32:42 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(ParseError), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 22:32:45 --> syntax error, unexpected 'resources' (T_STRING), expecting ')'
#0 D:\xampp\htdocs\ushno-app\system\View\View.php(222): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#1 D:\xampp\htdocs\ushno-app\system\Common.php(1217): CodeIgniter\View\View->render('emisora/pages/i...', Array, true)
#2 D:\xampp\htdocs\ushno-app\app\Controllers\HomeController.php(13): view('emisora/pages/i...')
#3 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(936): App\Controllers\HomeController->index()
#4 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(432): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\HomeController))
#5 D:\xampp\htdocs\ushno-app\system\CodeIgniter.php(333): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 D:\xampp\htdocs\ushno-app\public\index.php(44): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-07-20 22:32:45 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(ParseError), 500)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(ParseError))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 22:41:42 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-07-20 22:42:56 --> Uncaught ErrorException: include(): Filename cannot be empty in D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php:281
Stack trace:
#0 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): CodeIgniter\Debug\Exceptions->errorHandler(2, 'include(): File...', 'D:\\xampp\\htdocs...', 281, Array)
#1 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(281): include()
#2 D:\xampp\htdocs\ushno-app\system\Debug\Exceptions.php(147): CodeIgniter\Debug\Exceptions->render(Object(CodeIgniter\Exceptions\PageNotFoundException), 404)
#3 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(CodeIgniter\Exceptions\PageNotFoundException))
#4 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
